docs/components/tabs.md

```diff
@@ -176,6 +176,10 @@ const tabChange = (hash: string): void => {
 </script>
 ```
 
+### anchorTabindex
+
+This prop allows setting a custom `tabindex` for the tab anchor element. It’s useful when passing a custom interactive element, like a link, through the [`anchor` slot](#anchor-panel), ensuring that only the slotted element is focusable by resetting the default anchor `tabindex`. Default value is `0`.
```

Comment:
The docs don't match the interface of the new prop. Here you're describing the default as a `number` `0` but the actual interface only allows for a string `'0'`